docs/reference/install-kubernetes.md

```diff
@@ -51,6 +51,8 @@ $ kubectl gadget version
 ```
 
 ## Installing in the cluster
+Before deploying make sure that kubernates cluster is deployed . Like minikube is running .\
+Steps to make minikube running - [minikube](https://minikube.sigs.k8s.io/docs/start/?arch=%2Fwindows%2Fx86-64%2Fstable%2F.exe+download)
```

Comment:
What about keeping it generic like this:

```suggestion

Before you begin, ensure that your Kubernetes cluster is up and running and that you can access it with `kubectl`.
```

?

Reply:
okay cool!